### DISCUSSION TITLE
Safe to purge transactions from the database?

### DISCUSSION THREAD

**Discussion by disconsented** [firefly-iii/firefly-iii#9316] [Firefly III] [answered]
Short question: Is it safe to purge the contents of the transaction table from the database manually?

Explanation:

I was looking at my firefly, and I noticed that a few of my accounts were _significantly_ off. I wasn't able to find any transactions for the year that could explain this slip. I'd decided to purge my transactions and reimport them. This _eventually_ failed, and, reimporting via the importer, reports _duplicate transactions_. These don't appear in the UI, but, I can find them in the database.

Would purging these be safe?

**Comment by JC5** [comment]
Yes, it's safe to purge them.

**Comment by disconsented** [comment, marked as answer]
That was a half solution, needed to purge from `transaction_journals` as well. Thanks!

**Reply by JC5** [reply]
Ah nice. There's a button for this in your profile page as well :)

**Reply by disconsented** [reply]
I'll have to check that out later, but, the purge transaction button that I found didn't catch these.
